kclvm/ast/src/lib.rs

```diff
@@ -7,11 +7,27 @@ pub mod pos;
 pub mod token;
 pub mod token_stream;
 pub mod walker;
+use std::sync::Mutex;
+
+use lazy_static::lazy_static;
 
 #[cfg(test)]
 mod tests;
 
-pub const MAIN_PKG: &str = "__main__";
+pub const MAIN_PKG_DEFAULT: &str = "__main__";
+
+lazy_static! {
```

Comment:
I am not sure if this method is feasible, as during the compilation process, there can be two threads simultaneously, one thread modifying main_pkg and the other thread compiling simultaneously, which will achieve the expected results.